week3/leap_year.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python
+
+'''
+This program first prompts the user to enter a year
+as an int value and checks if it is a leap year.
+
+>>> is_leap_year(2000)
+True
+'''
+
+def is_leap_year(year):
+
+    return True if year%4 == 0 and year%100 != 0 or year%400 == 0 else False
```

Comment:
good even if this is more pythonic:
```
if year%4 == 0 and year % 100 != 0 or year % 400 == 0:
    return True
else:
    return False
```
also notice the space between operator (%)